enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -46,11 +47,16 @@
      (s/optional-key :mode)     Mode}
     (deferred-trs "invalid context seed value")))
 
+(defn- check-premium-token! []
+  (when-not (premium-features/has-feature? :serialization)
+    (throw (Exception. (trs "Serialization requires a premium token with the serialization feature.")))))
```

Comment:
Could we make this error message a little more user-friendly? If you try to use SerDes and see this it might not mean a lot to someone who doesn't know how our token feature flags work internally

Reply:
I've rephrased it. Happy to change it again if we can think of some better phrasing.